packages/minuteman/extra/dcos-minuteman.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Layer 4 Load Balancer: DC/OS Layer 4 Load Balancing Service
+
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+WorkingDirectory=/opt/mesosphere/active/minuteman/minuteman
+EnvironmentFile=/opt/mesosphere/etc/minuteman-erl.env
+EnvironmentFile=/opt/mesosphere/etc/check_time.env
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/minuteman.env
+EnvironmentFile=-/run/dcos/etc/minuteman_auth.env
+ExecStartPre=/opt/mesosphere/bin/check-time
+ExecStartPre=/usr/bin/env modprobe ip_vs_wlc
```

Comment:
Running modprobe explicitly is caught by things like selinux, and results in hard failure. Implicitly loading the module because you asked for an API which needs it is not